lib/github_heroku_deployer/git.rb

```diff
@@ -12,17 +12,18 @@ def initialize(options)
       @repo_dir = options[:repo_dir]
     end
 
-    def push_app_to_heroku(remote="heroku", branch="master", &block)
+    def push_app_to_heroku(remote="heroku", branch="update-gemfilelock", &block)
```

Comment:
maybe leave this defaulted to `master` but have the caller call the `update-gemfilelock` as branch